lib/benchee/system.ex

```diff
@@ -37,8 +37,8 @@ defmodule Benchee.System do
       {:ok, version} ->
         String.trim(version)
 
-      {:error, reason} ->
-        IO.puts("Error trying to determine erlang version #{reason}")
+      {:error, _reason} ->
+        to_string(otp_release)
```

Comment:
The previous error message was:

`Error trying to determine erlang version ENOENT`

With this PR it is changed to:

`23` (or whichever version `:erlang.system_info(:otp_release)` returns.

imho this proposed error message is not more helpful.

Reply:
I think the intent here was not to return `nil` and instead return just the full OTP release, no error message as to avoid the crash.

We can do that, but I think I still wanna print the warning. Will adapt.